File: src/main/java/com/tom/student/Student.java
package com.tom.student;
public class Student {
    String id;
    String name;
    int english;
    int math;

    public Student(String name, int english, int math) {
        this.name = name;
        this.english = english;
        this.math = math;
    }

    public int hightest(){
        /*int max = ;
        if (english > math){
            max = english;
        } else {
            max = math;
        }*/
        return (english > math)? english : math;
    }

    public void print(){
        System.out.print(name +"\t"+ english +"\t"+ math
                +"\t"+ getAversge());
        if (getAversge() >= 60){
            System.out.println("\tPASS");
        } else {
            System.out.println("\tFAILED");
        }
    }

    public int getAversge (){
        return (english+math)/2;
    }
}

File: src/main/kotlin/com/tom/student.kt
package com.tom

import java.util.Scanner

fun main() {
//    userInput()

    val stu = Student("Zoey", 52, 74)
    stu.print()
    println("High score : ${stu.hightest()}")

    var billy = 123
    println("Test billy : $billy")

    /*val stu = Student("Zoey", 52, 74)
    stu.print()*/
}

class Student(var name:String?, var english:Int, var math:Int){
    fun print(){
        println(name +"\t"+ english +"\t"+ math
                +"\t"+ getAversge() + "\t"
                + if (getAversge() >= 60) "PASS" else "FAILED" )
    }

    fun getAversge ():Int{
        return (english+math)/2
    }

    fun hightest() : Int {
//        var max = if (english > math) english else math
        var max = if (english > math){
            println("english")
            english
        } else {
            println("math")
            math
        }
        return max
    }

private fun userInput() {
    val scanner = Scanner(System.`in`)
    print("Please enter student's name:\n")
//    var name = scanner.next()
    var name = null
    print("Please enter student's english:")
    var english = scanner.nextInt()
    print("Please enter student's math:")
    var math = scanner.nextInt()


    val stu = Student(name, english, math)
    stu.print()
    stu.checkName()
}

    fun checkName(){
        println(name?.length)
    }
}